src/App.js

```diff
@@ -14,11 +15,100 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [...goodsFromServer],
+    buttonVisible: true,
+  }
+
+  showList = () => {
+    this.setState(state => ({
+      buttonVisible: false,
+    }));
+  }
+
+  reverseList = () => {
+    this.setState(state => ({
+      goods: [...state.goods].reverse(),
+    }));
+  }
+
+  sortAlpbList = () => {
+    this.setState(state => ({
+      goods: [...state.goods].sort(),
+    }));
+  }
+
+  sortLengthList = () => {
+    this.setState(state => ({
+      goods: [...state.goods].sort((a, b) => b.length - a.length),
+    }));
+  }
+
+  resetList = () => {
+    this.setState(state => ({
+      goods: [...goodsFromServer],
+    }));
+  }
+
+  render() {
+    const { goods } = this.state;
+
+    this.a = 10;
```

Comment:
What is this for?

Reply:
Sorry, was checking smth, forgot to delete.